GitUtils.ps1

```diff
@@ -313,7 +313,13 @@ function Start-SshAgent([switch]$Quiet) {
             }
         }
     }
-    Add-SshKey
+    $keyfiles = Get-ChildItem -Path "$home/.ssh/*" -Exclude *.*, known_hosts
+    $keyfilepaths = @()
+    foreach($f in $keyfiles)
+    {
+        $keyfilepaths = $keyfilepaths + $f.FullName -replace "c:", "/c" -replace "\\", "/" 

```

Comment:
I haven't found this sort of path manipulation to be necessary with MinGW, and [that seems to be one of their goals](http://sourceforge.net/p/mingw/mailman/message/8686481/).